Sources/Starknet/Data/TypedData/StarknetTypedData.swift

```diff
@@ -384,7 +384,7 @@ public struct StarknetTypedData: Codable, Equatable, Hashable {
 
     public func getMessageHash(accountAddress: Felt) throws -> Felt {
         try hashArray([
-            Felt.fromShortString("StarkNet Message")!,
+            Felt.fromShortString("Starknet Message")!,
```

Comment:
> The `PREFIX_MESSAGE` **must be** `StarkNet Message`.

per the [SNIP](https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-12.md#prefix-message). 

Let's verify all changes to `StarkNet` spelling in typed data and make sure we should actually make them